packages/frontend/src/features/sqlRunner/components/Tables.tsx

```diff
@@ -0,0 +1,58 @@
+import { LoadingOverlay, Stack, Text, UnstyledButton } from '@mantine/core';
+import { useState, type FC } from 'react';
+import { useTables } from '../hooks/useTables';
+
+type Props = {
+    projectUuid: string;
+};
+
+export const Tables: FC<Props> = ({ projectUuid }) => {
```

Comment:
[not blocker] Would be nice to have a search for the tables.

Reply:
Nice! Will add search functionality to this and the fields on a separate PR

Reply:
added here: https://github.com/lightdash/lightdash/issues/10569